Frontend/CO.CDP.OrganisationApp/Pages/Organisation/OrganisationOverview.cshtml.cs

```diff
@@ -21,6 +27,16 @@ public async Task<IActionResult> OnGet()
         try
         {
             OrganisationDetails = await organisationClient.GetOrganisationAsync(Id);
+
+            if (OrganisationDetails.IsBuyer() || OrganisationDetails.IsPendingBuyer())
+            {
+                BuyerInformation = await organisationClient.GetOrganisationBuyerInformationAsync(OrganisationDetails.Id);
```

Comment:
Just wanted to double check: If they're pending buyer, is the buyer information actually available?

Reply:
Double checked. Yes .The buyer information is available for a pending buyer.